pkg/oidcclient/login.go

```diff
@@ -515,12 +524,14 @@ func (h *handlerState) cliBasedAuth(authorizeOptions *[]oauth2.AuthCodeOption) (
 }
 
 // Prompt for the user's username and password, or read them from env vars if they are available.
-func (h *handlerState) getUsernameAndPassword() (string, string, error) {
+func (h *handlerState) getUsernameAndPassword(out io.Writer) (string, string, error) {
 	var err error
 
+	_, _ = fmt.Fprintf(out, "Log in to %s:\n", h.upstreamIdentityProviderName)
```

Comment:
```suggestion
	_, _ = fmt.Fprintf(out, "\nLog in to %s\n\n", h.upstreamIdentityProviderName)
```

Suggesting to make this look more like a login banner than an interactive prompt by adding extra newlines and removing the colon.

This would make your terminal session look like this:

```
❯ pinniped whoami --kubeconfig ./kubeconfig-ldap.yaml

Log in to My LDAP IDP 🚀

Username: pinny
Password:
Current cluster info:

Name: kind-pinniped-pinniped
URL: https://127.0.0.1:59524

Current user info:

Username: ldap:pinny.ldap@example.com
Groups: ldap:ball-admins, ldap:ball-game-players, system:authenticated
```

instead of this, which is more cramped and makes it look like a prompt (due to the colon):

```
❯ pinniped whoami --kubeconfig ./kubeconfig-ldap.yaml
Log in to My LDAP IDP 🚀:
Username: pinny
Password:
Current cluster info:

Name: kind-pinniped-pinniped
URL: https://127.0.0.1:59524

Current user info:

Username: ldap:pinny.ldap@example.com
Groups: ldap:ball-admins, ldap:ball-game-players, system:authenticated
```